tests/meilisearch_helper/version_test.py

```diff
@@ -0,0 +1,9 @@
+# coding: utf-8
+import re
+
+from scraper.src.config.version import __version__
+
+
+class TestInit:
+    def test_get_version(self):
+        assert re.match(r"^(\d+\.)?(\d+\.)?(\*|\d+)$", __version__)
```

Comment:
what are we trying to test here?

Reply:
That the version is properly formatted as: `x.x.x` (where x is digit) However it's not mandatory but as the same exists I thought it wouldn't hurt to test the version format here too.